<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlPanel</class>
 <widget class="QFrame" name="ControlPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>179</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Plain</enum>
  </property>
  <property name="lineWidth">
   <number>0</number>
  </property>
  <widget class="QTabWidget" name="m_panel_settings">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>7</y>
     <width>655</width>
     <height>158</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::South</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="m_tab_settings">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <attribute name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/images/show.svg</normaloff>:/images/show.svg</iconset>
    </attribute>
    <attribute name="title">
     <string>Settings</string>
    </attribute>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <item row="1" column="2">
      <widget class="QLabel" name="label_parity">
       <property name="text">
        <string>Parit&amp;y</string>
       </property>
       <property name="buddy">
        <cstring>m_combo_parity</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QComboBox" name="m_combo_parity"/>
     </item>
     <item row="2" column="3">
      <widget class="QComboBox" name="m_combo_stopBits"/>
     </item>
     <item row="0" column="3">
      <widget class="QComboBox" name="m_combo_dataBits"/>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_stopBits">
       <property name="text">
        <string>Stop Bits</string>
       </property>
       <property name="buddy">
        <cstring>m_combo_stopBits</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QCheckBox" name="m_check_lineBreak">
       <property name="text">
        <string>Display &amp;Ctrl characters</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QCheckBox" name="m_check_timestamp">
       <property name="text">
        <string>Show &amp;Timestamp</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="m_combo_flowControl"/>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="m_combo_Baud"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_baud">
       <property name="text">
        <string>&amp;Baudrate</string>
       </property>
       <property name="buddy">
        <cstring>m_combo_Baud</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_dataBits">
       <property name="text">
        <string>D&amp;ata Bits</string>
       </property>
       <property name="buddy">
        <cstring>m_combo_dataBits</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_mode">
       <property name="text">
        <string>Open &amp;Mode</string>
       </property>
       <property name="buddy">
        <cstring>m_combo_Mode</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_flowControl">
       <property name="text">
        <string>Flo&amp;w Control</string>
       </property>
       <property name="buddy">
        <cstring>m_combo_flowControl</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QFrame" name="logging_frame">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="m_lb_logfile">
          <property name="text">
           <string>Logfile:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="m_logfile_edit">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>400</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="m_bt_logfileDialog">
          <property name="toolTip">
           <string>Choose Logfile via file chooser.</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="m_check_appendLog">
          <property name="toolTip">
           <string>Appends to an existing logfile instead of truncating it.</string>
          </property>
          <property name="text">
           <string>Append</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="hSpacer_settings_grid">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::MinimumExpanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>2</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="m_combo_Mode"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="m_device_control_bar">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>140</y>
     <width>561</width>
     <height>36</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <item>
     <widget class="QPushButton" name="m_bt_open">
      <property name="text">
       <string>&amp;Open</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_device">
      <property name="text">
       <string>De&amp;vice:</string>
      </property>
      <property name="buddy">
       <cstring>m_combo_device</cstring>
      </property>
     </widget>
    </item>
    <item>
     <widget class="DeviceCombo" name="m_combo_device" native="true"/>
    </item>
    <item>
     <widget class="QCheckBox" name="m_rts_line">
      <property name="toolTip">
       <string>The RTS line is set to logic HIGH when checked. Note that in RS232C the negative logic is used.</string>
      </property>
      <property name="text">
       <string>RTS</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="m_dtr_line">
      <property name="toolTip">
       <string>The DTR line is set to logic HIGH when checked. Note that in RS232C the negative logic is used.</string>
      </property>
      <property name="text">
       <string>DTR</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DeviceCombo</class>
   <extends>QWidget</extends>
   <header>devicecombo.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_combo_Baud</tabstop>
  <tabstop>m_combo_flowControl</tabstop>
  <tabstop>m_combo_Mode</tabstop>
  <tabstop>m_combo_dataBits</tabstop>
  <tabstop>m_combo_parity</tabstop>
  <tabstop>m_combo_stopBits</tabstop>
  <tabstop>m_check_lineBreak</tabstop>
  <tabstop>m_check_timestamp</tabstop>
  <tabstop>m_logfile_edit</tabstop>
  <tabstop>m_bt_logfileDialog</tabstop>
  <tabstop>m_check_appendLog</tabstop>
  <tabstop>m_combo_device</tabstop>
  <tabstop>m_bt_open</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
